dart/lib/src/sentry_options.dart

```diff
@@ -254,5 +255,10 @@ void noOpLogger(SentryLevel level, String message) {}
 
 /// A Logger that prints out the level and message
 void dartLogger(SentryLevel level, String message) {
-  print('[${level.name}] $message');
+  log(
+    '[${level.name}] $message',
+    level: level.toLogLevel(),
+    name: '[sentry]',
```

Comment:
```suggestion
    name: 'sentry',
```

`log` also add brackets